STEERING-COMMITTEE.md

```diff
@@ -37,8 +38,6 @@ the way we run this committee, based on feedback from the community.
    disposition of those resources.
 1. Manage the Knative brand and decide which things can be called "Knative" and
    how that mark can be used in relation to other efforts or vendors.
-1. Confirm/reject nominations to the KSC from organizations who are allocated
-   seats.
 1. Confirm/reject nominations to the Technical Oversight Committee.
```

Comment:
Just to play devil's advocate a little here: why should the Steering Committee have the ability to reject a nomination to the TOC when the TOC is elected on essentially the same basis, from just as representative a pool, as the Steering committee? (in fact, almost by definition a TOC election being considered by the Steering Committee would be _more_ representative, in that it would obviously be more recent if it were being confirmed or rejected by the existing body).

Reply:
I think better wording here now would be "Serve as TOC election officers" (linking to election mechanics), since I believe the TOC election mechanics say something similar.  

Happy to discuss that detail in more depth (I had the same thought), but I don't want to detract from the substance of the PR.  😇

Reply:
Yep, if this becomes just an election officer responsibility rather than this confirm/reject wording that seems fine. 

I do think though fwiw that this actually cuts quite directly at a major hesitation I have about this charter (which generally I support) in a more general sense. How does having both TOC and Steering elected change the relationship between those bodies? For example this document defines the TOC as being a delegation of the Steering committee, presumably implying steering can override TOC decisions, or abolish the TOC.. is that still reasonable when both are elected bodies? Why? In what sense does, should or can Steering posses greater legitimacy?

Reply:
re: combining

I think the groups ultimately cater to different candidate-audiences (perhaps overlapping) because they generalize exercise different skills operating in different aspects of the project.

I could see combining them dissuading candidates from joining the body, or elections for a single body resulting in a bias towards one skillset and a deficit of the other.  Don't ask me to walk and chew gum, @julz! 😉 

---

re: hierarchy

I think some of this wording is prehistoric and should probably be revisited.  Ideally steering (as election officers) would be responsible for tweaking election mechanics, and it's a good question whether more extreme capabilities are still needed.

Personally, I would mind this clause less if Steering were also elected from the community 😇

Reply:
To state the somewhat obvious, the problem with two committees _without_ hierarchy is that you have no obvious mechanism for how to resolve conflicts, and conflicts happen. The problem with two committees _with_ hierarchy when both are elected from essentially the same pool of candidates and electors it there's a very clear legitimacy problem when one overrules the other, particularly if the "junior" committee happens to've been more recently elected.

Reply:
(for the record I 100% believe in your ability to both walk and chew gum @mattmoor, but I of course respect any decision you make not to ;))

Reply:
> Yep, if this becomes just an election officer responsibility rather than this confirm/reject wording that seems fine.
> 
> I do think though fwiw that this actually cuts quite directly at a major hesitation I have about this charter (which generally I support) in a more general sense. How does having both TOC and Steering elected change the relationship between those bodies? For example this document defines the TOC as being a delegation of the Steering committee, presumably implying steering can override TOC decisions, or abolish the TOC.. is that still reasonable when both are elected bodies? Why? In what sense does, should or can Steering posses greater legitimacy?

Seems like the last bullet point in this charter is what's being discussed here? The way I read this:
```
1. Act as the final escalation point and decider for any disputes, issues,
   clarifications, or escalations within the project scope.	   clarifications, or escalations within the project scope.
```

Is that if TOC makes a decision, it gets escalated to SC they would have the authority to override that decision?

Reply:
To agree with julz's point, above, there needs to be a clear ultimate authority. I would prefer that authority rest with the community part of the committees, rather than the technology part of the committees. In particular, the Steering Committee is the group which adjudicates Code of Conduct complaints, and I'd prefer that those decisions to override *any* technical considerations.

Reply:
Just a note: we decided to have a distinct discussion about the relationship of the SC/TOC, dev list thread here:

https://groups.google.com/g/knative-dev/c/AOm2exWqjJg

Reply:
I decided the language about approving/rejecting nominations doesn't make a lot of sense, I am going to strike it.